core/src/epicli/data/common/ansible/playbooks/roles/upgrade/tasks/rabbitmq/check-version-redhat.yml

```diff
@@ -0,0 +1,4 @@
+- name: RabbitMQ | Check RedHat package version
+  shell: yum list rabbitmq-server | tail -1 | awk '{print $2}'
```

Comment:
Would this be possible to use here `package_facts` https://docs.ansible.com/ansible/latest/collections/ansible/builtin/package_facts_module.html ? :thinking: